R/description.R

```diff
@@ -19,7 +19,8 @@
 #' ```
 #' options(
 #'   usethis.description = list(
-#'     `Authors@R` = 'person("Jane", "Doe", email = "jane@example.com", role = c("aut", "cre"))',
+#'     `Authors@R` = 'person("Jane", "Doe", email = "jane@example.com", role = c("aut", "cre"),
+#'                           comment = c(ORCID = "YOUR-ORCID-ID"))',
```

Comment:
A bit nit-picky, sorry, but is the duplication of `ID` intended here & in the changes below?

Reply:
You mean ORCID ID instead of ORCID? That's what ORCID writes https://orcid.org/register although I see it should be iD